config/api.yml

```diff
@@ -461,6 +461,9 @@
         :identifier: miq_report_run
     :subcollections:
     - :results
+    :collection_actions:
+      :post:
+      - :name: import
```

Comment:
@abellotti i couldn't find an appropriate identifier for importing reports here - the closest i could find was `miq_report_export` which still allowed my test to pass with `api_basic_authorize`. may need some pointers here